src/invoices/InvoiceDetails/InvoiceActions/InvoiceActions.js

```diff
@@ -28,6 +28,7 @@ const InvoiceActions = ({
   onPay,
   onPrint,
   onInvoiceCancel,
+  isApprovePayButtonDisabled = false,
```

Comment:
we have `isApprovePayEnabled`, I would suggest to rename it to `isApprovePayAvailable` to show/hide action and `isApprovePayButtonDisabled` -> `isApprovePayEnabled`

Reply:
` = false` defaultProps is defined for this purpose